pages/about_new.tsx

```diff
@@ -149,7 +148,16 @@ const WhatWeDoSection = ({ theme }) => (
 const OurValueSection = ({ theme }) => (
   <AboutUsSection backgroundColor={theme.palette.background.default}>
     <Typography variant="headlineMedium">Our values</Typography>
-    <CardRowContainer>
+    <Box
+      sx={{
+        display: 'grid',
+        gridTemplateColumns: {
+          xs: '1fr',
+          lg: 'repeat(3, minmax(0, 1fr))',
```

Comment:
what does this mean? `'repeat(3, minmax(0, 1fr))'`

Reply:
I want equal width columns, and I had been using `repeat(3, 1fr)`, but that still resulted in columns of different widths which looked a little weird! So I found this line from this [stackoverflow](https://stackoverflow.com/questions/47601564/equal-width-columns-in-css-grid) if you wanna look :)